Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1185,24 +1200,30 @@ func _vjpLeakyRelu<T: TensorFlowFloatingPoint>(
     _ x: Tensor<T>,
     alpha: Double
 ) -> (Tensor<T>, (Tensor<T>) -> Tensor<T>) {
-    return (leakyRelu(x, alpha: alpha), { v in
+    (leakyRelu(x, alpha: alpha), { v in
         Raw.leakyReluGrad(gradients: v, features: x, alpha: alpha)
     })
 }
 
-/// Returns a tensor by applying the ReLU activation function to the specified tensor element-wise.
-/// Specifically, computes `max(0, x)`.
+/// Returns `scale * alpha * (exp(x) - 1)` if `x < 0`, and `scale * x` otherwise.
```

Comment:
Just a small suggestion in accordance with the rest of the activation function docs.

```suggestion
/// Returns  a tensor by applying the SeLU activation function to the specified tensor element-wise.
/// Specifically, computes `scale * alpha * (exp(x) - 1)` if `x < 0`, and `scale * x` otherwise.
```

Reply:
That's a good observation. I generally tend towards less verbosity in documentation strings (while not sacrificing clarity) and so would suggest changing the other activation function doc strings to be more concise instead. @Shashi456 @rxwei what do you think?

Reply:
How about:

> Returns a tensor by applying the SeLU activation function, namely `scale * alpha * (exp(x) - 1)` if `x < 0`, and `scale * x` otherwise.

Reply:
I'm positive about mentioning "the SeLU activation function" somewhere, which is a term that people can google to find the exact definition. Googling just the function name "selu" will not give an exact definition.

Reply:
The adherence to this format was introduced in #160.

Reply:
Sounds good. I'll make the change.